### ISSUE TITLE
Crashed when loading a chunk and can no longer load in without server crash

### ISSUE THREAD

**Issue by Supercow1122** [TelepathicGrunt/RepurposedStructures-Fabric#200]
As it states in the subject. I can provide the crash log if needed. Another thing is im using Another Quality Modpack 2 and i updated my server from 1.18.1 to 1.18.2 version of minecraft and im wondering if thats why this is happening. I have had other crashes that seem to have to do with Repurposed Structures in terms of which building is "causing" it. First was a witch hut and then now the latest is a nether pyramid. Any help would be appreciated!

**Comment by TelepathicGrunt** [comment]
Basically a duplication of this issue: https://github.com/TelepathicGrunt/RepurposedStructures-Fabric/issues/199


Tldr: it’s really not safe to try and upgrade heavily modded worlds. Especially with DFU quite hostile to modded data every update lol.

The chunks in the nether and such are saved with the old 1.18.1 repurposed structures data that no longer exists in 1.18.2 as I did a big rewrite in 1.18.2. I can’t really prevent the issue because this is outdated data stuck in chunks. You can delete the nether to regenerate it and that will fix the crash in nether. However, I cannot say the same for other dimensions as I do not remember if I changed the processor names and paths for structures in overworld/end. Since you mentioned a witch hut, then I guess I changed that processor too. 

Best thing to do is really to restart the world or try and create a datapack that holds an empty processor list for each crash that tries to get a non-existent 1.18.1 processor
